member_map_internal_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 DE-labtory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cleisthenes
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestMemberMap_Add(t *testing.T) {
+	memberMap := NewMemberMap()
```

Comment:
Why `Add`, `Del` to internal test? all these are public.

how about move this code into member_map_test.go ?